internal/netsync/manager.go

```diff
@@ -1701,6 +1701,14 @@ func (m *SyncManager) IsCurrent() bool {
 	return isCurrent
 }
 
+// RecentlyConfirmedTxn returns with high degree of confidence whether a
+// transaction has been recently confirmed in a block.
+//
+// This method may report a false positive, but never a false negative.
+func (m *SyncManager) RecentlyConfirmedTxn(hash *chainhash.Hash) bool {
```

Comment:
I don't think this should be exported from here.  The server is the thing that owns it and the RPC adaptor can just use the one on server directly.  See my other comment.